### PR TITLE
Remove incentivized channel, apps, channel ID & sundry from contracts

### PR DESCRIPTION
**Pull request by doubledup**
`pnpm build` & `pnpm test` pass. I've left `ChannelAccess` in for now, not sure if we'll still use it for the channel. Happy to remove it if we won't.

After #753 and #756, this is the last big piece for removing the incentivized channel & apps.

### CODE REVIEW

File: core/packages/contracts/.prettierrc [hunk 1]
```diff
@@ -7,7 +7,6 @@
           "tabWidth": 4,
           "singleQuote": false,
           "bracketSpacing": true,
-          "explicitTypes": "always",
```
Comment (claravanstaden) [inline core/packages/contracts/.prettierrc:10]: Why do we need to remove this?

**Review by claravanstaden — APPROVED** [review]
Great! These smaller PRs are very digestible, thanks for splitting them up. 😅

**Review by vgeddes — COMMENTED** [review]
+1

File: core/packages/contracts/.prettierrc (hunk 1 above)
Comment (doubledup) [reply core/packages/contracts/.prettierrc:10]: It was deprecated in `prettier-plugin-solidity` 1.0.0. See the body of 4b3565d for details.

File: core/packages/contracts/deploy/03-channels.ts [hunk 2]
```diff
@@ -23,64 +20,11 @@ module.exports = async ({ ethers, deployments, getUnnamedAccounts, network }: Ha
         maxPriorityFeePerGas: feeData.maxPriorityFeePerGas,
     })
 
-    let incentivizedInboundChannel = await deployments.deploy("IncentivizedInboundChannel", {
+    await deployments.deploy("BasicOutboundChannel", {
         from: deployer,
-        args: [config.incentivizedChannelSourceID, parachainClient.address],
         log: true,
         autoMine: true,
```
[the remark this answers is not in the document]
Comment (doubledup) [reply core/packages/contracts/deploy/03-channels.ts:26]: I think we can move them to the hardhat config. Testing it locally

[the remark this answers is not in the document]
Comment (doubledup) [reply core/packages/contracts/deploy/03-channels.ts]: Some of my integration tests are failing locally on main, will debug next week.

[the remark this answers is not in the document]
Comment (doubledup) [reply core/packages/contracts/deploy/03-channels.ts]: Next thing is to get the tests passing, so I'll add this to those changes and merge this now.

**Pull request merged**